tests/testSed.py

```diff
@@ -139,12 +152,12 @@ def test_read_sed_flambda(self):
         Test how readSED_flambda handles the reading of SED filenames
         when we fail to correctly specify their gzipped state.
         """
-        scratch_dir = os.path.join(getPackageDir("sims_photUtils"),
-                                   "tests", "scratchSpace")
```

Comment:
I just wanted the ease of using `teardown_module` to delete all of the temporary files created in this script.  If you would rather, I can do as you suggest and use `tearDownClass` (or just `tearDown`) to delete the scratch dir.